templates/compute-domain-daemon.tmpl.yaml

```diff
@@ -37,6 +37,11 @@ spec:
         # Use runc: explicit "void"; otherwise we inherit "all".
         - name: NVIDIA_VISIBLE_DEVICES
           value: void
+        {{- if .FeatureGates }}
+        # Feature gates (includes both project-specific and logging features)
+        - name: FEATURE_GATES
+          value: "{{ range $key, $value := .FeatureGates }}{{ $key }}={{ $value }},{{ end }}"
+        {{- end }}
```

Comment:
So, we decide to set feature flags via environment variables. I like that.

How would one set a feature flag via (a) CLI argument(s)?